sylvia/tests/replies.rs

```diff
@@ -77,9 +79,13 @@ mod reply_contract {
 fn entry_point_generation() {
     let msg = Reply {
         id: 0,
+        payload: Default::default(),
+        gas_used: 0,
+        #[allow(deprecated)]
```

Comment:
Is there a way to update this test without adding `allow(deprecated)`?

Reply:
I think there is not. There is an issue about it https://github.com/rust-lang/rust/issues/102777, but no action so far.